### PR TITLE
feat(chat): support "\n \n" delimiter for model description (Issue #2332)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

support "\n \n" delimiter for model description

Issues:

- Issue #2332

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2333.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Derikyan — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2333.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/components/Chat/ModelList.tsx [hunk 1]
```diff
@@ -257,7 +257,8 @@ const ModelGroup = ({
         </div>
         {!notAllowExpandDescription &&
           description &&
-          description.indexOf('\n\n') !== -1 && (
+          (description.indexOf('\n\n') !== -1 ||
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/ModelList.tsx]: what do you think about regex for this case? to handle such cases when we have more than 1,2,3 and more spaces between

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/525258)

**Review by Alexander-Kezik — APPROVED** [review]

**Review by irinakartun — APPROVED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2333.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**